README.md

```
@@ -1 +1,151 @@
+   | Budget Services | CRUD operations on expense, create/ update savings, create/update salary |
+   | Mail Services | Sends the update via mails |
+   | Report Services | Generates the report on monthly weekly, yearly basis |
+
+  ---
+  <div>
+    <h3>4. Project source code link 👇 </h3>
+  </div>
+
+  |Title | Code Link |
+  | ---- | --------- |
+  |View  | <a href="https://github.com/Shwetha-75/Budget-Tracker-View.git" target="_blank">View</a>
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)